examples/basic/package.json

```diff
@@ -3,11 +3,8 @@
   "scripts": {
     "build": "pulp browserify --optimise --to dist/example.js"
   },
-  "dependencies": {
-    "virtual-dom": "^2.1.1"
-  },
   "devDependencies": {
     "pulp": "^10.0.0",
-    "purescript": "^0.10.2"
+    "purescript": "^0.10.5"
```

Comment:
Yeah I guess that makes sense. Also all of the examples aside from `ace` don't need to use `pulp browserify` either, `build` with `--to` suffices, so I'll update those too.